.rubocop.yml

```diff
@@ -5,4 +5,5 @@ inherit_gem:
 
 AllCops:
   Exclude:
-    - "test/tmp/**/*"
+    - test/tmp/**/*
+    - vendor/bundle/**/*
```

Comment:
Not sure if it was intentional, but it looks like the trailing newline was removed:

![image](https://user-images.githubusercontent.com/1863540/91751748-29ea9b80-eb7a-11ea-8266-324fdfc2bd71.png)